secretsanta/main/utils.py

```diff
@@ -0,0 +1,8 @@
+import logging
+import time
+import os
+def setup_logging(level: str = "ERROR"):
+    name_file = 'secretsanta_' + time.strftime("%Y%m%d-%H%M%S") +'.log'
+    path_file = os.path.join('./log_files/', name_file)
+    return logging.basicConfig(filename =path_file, level = level, format = '%(asctime)s %(levelname)s %(message)s',
```

Comment:
it seems you are not assigning a return value when calling this function?

Reply:
I assume you actually didn't want to return anything. It is good that you have the type-hint - possibly we should also add a minimal docstring?

Reply:
return statement has been removed